erc20-price/src/lib.rs

```diff
@@ -111,16 +114,14 @@ fn store_chainlink_aggregator(block: eth::Block, output: StoreSetProto<Aggregato
             let aggregator = Aggregator {
                 address: Hex(&aggregator_address).to_string(),
                 description: description.clone(),
-                base: base_quote[0].to_string(),
-                base_address: base_address.clone(),
-                quote: base_quote[1].to_string(),
-                quote_address: quote_address.clone(),
+                base_asset: utils::get_erc20_token(base_address),
```

Comment:
it's gonna be very expensive calling this every time. Since this data doesn't change, we should have a separate module to retrieve this data so we don't need to query it over and over again